### ISSUE TITLE
The /*comment*/ syntax causes all SCSS afterwards to not be included

### ISSUE THREAD

**Issue by chrismbarr** [madskristensen/WebCompiler#92]
With the most recent update (v 1.3.136) I've noticed that as soon as any SCSS with a multi-line comment is encountered, any following code does not seem to be included.

:computer: **compilerconfig.json** setup

```json
[
	{
		"inputFile": "sass/test.scss",
		"outputFile": "sass/test.css"
	}
]
```

:arrow_forward: **SCSS input:**
```SCSS
#test1{
	color: red;
}

/*#test2{
	color: blue;
}*/

#test3{
	color: green;
}
```

:x: **Incorrect CSS Output:** (missing the `#test3` selector that should appear after the comment)
```CSS
#test1 {
  color: red; }

/*#test2{
	color: blue;
}*/
```

However, switching to the single-line comments will fix this, so this is my temporary workaround
:arrow_forward: **SCSS input:**
```SCSS
#test1{
	color: red;
}

//#test2{
//	color: blue;
//}

#test3{
	color: green;
}
```
:white_check_mark: **Correct CSS Output:** (Comments not included, and only uncommented code is output)
```CSS
#test1{
	color: red; }

#test3{
	color: green; }
```

**Comment by madskristensen** [comment]
Please open a bug on the node-sass library which is what Web Compiler is using. https://github.com/sass/node-sass/issues

**Comment by chrismbarr** [comment]
Ok thanks, I've opened a bug with the node-sass library now: https://github.com/sass/node-sass/issues/1118

**Comment by chrismbarr** [comment]
They say they cannot reproduce this bug, could it be possible that it is an issue with WebCompiler after all?

**Comment by madskristensen** [comment]
I doubt it, but I'll investigate.

**Comment by madskristensen** [comment]
It was indeed my code. It had to do with the way I handle source map comments. Can you please help verify the fix in the CI build version 1.3.140 or newer at http://vsixgallery.com/extension/148ffa77-d70a-407f-892b-9ee542346862/

**Comment by chrismbarr** [comment]
Great, glad it was an easy fix.  I installed version 1.3.143 and reverted to using multi-line comments and everything is working as expected now.  Thanks for looking into this!
